pkg/kubelet/config/file_linux.go

```diff
@@ -80,7 +84,7 @@ func (s *sourceFile) watch() error {
 	}
 }
 
-func (s *sourceFile) processEvent(e *inotify.Event) error {
+func (s *sourceFileListerWatcher) processEvent(e *inotify.Event) error {
```

Comment:
We should handle  IN_ATTRIB too, as described in the issue https://github.com/kubernetes/kubernetes/issues/40123